src/main/java/org/cyclonedx/generators/xml/BomXmlGenerator10.java

```diff
@@ -59,17 +59,19 @@ public Version getSchemaVersion() {
      * @since 1.1.0
      */
     public Document generate() throws ParserConfigurationException {
-        final DocumentBuilder docBuilder = buildSecureDocumentBuilder();
-        this.doc = docBuilder.newDocument();
-        doc.setXmlStandalone(true);
-
-        // Create root <bom> node
-        final Element bomNode = createRootElement("bom", null,
-                new Attribute("xmlns", NS_BOM_10),
-                new Attribute("version", String.valueOf(bom.getVersion())));
+        try {
+            return generateDocument(this.bom);
+        }
+        catch (XMLStreamException e) {
+            e.printStackTrace();
+        }
+        catch (IOException e) {
+            e.printStackTrace();
+        }
```

Comment:
Likely can collapse these and throw the ParserConfigurationException (or something akin)

Reply:
Donezo